README.md

```diff
@@ -16,6 +16,20 @@ It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controlle
 
 This project is scaffolded using [Kubebuilder](https://book.kubebuilder.io), and all the Kubebuilder [`makefile` helpers](https://book.kubebuilder.io/reference/makefile-helpers.html) can be used.
 
+## Install
+
```

Comment:
Are these instructions alternatives to each other, or subsequent steps?
I suspect they are both required, not alternatives. 
In that case, use a numbered list and add indentations to the code blocks so they're properly aligned below the respective step.

Reply:
They are required steps steps.